internal/monitor/exporter.go

```diff
@@ -15,11 +15,14 @@
 package monitor
 
 import (
+	"context"
 	"fmt"
+	"net/http"
 	"strings"
 	"time"
 
 	"contrib.go.opencensus.io/exporter/ocagent"
+	"contrib.go.opencensus.io/exporter/prometheus"
```

Comment:
I choose to use this archived repo to make the exporters consistent as other exporters are also using the same archived lib, for example `contrib.go.opencensus.io/exporter/stackdriver`.

I am working on a separate commit to try to migrate the `contrib.go.opencensus.io` lib to `[opentelemetry-go](https://github.com/open-telemetry/opentelemetry-go)`, but it may take longer.

Reply:
After some attempts to refactor and migrate, I realize that all the existing metrics exporters rely on `internal/monitor/reader.go` and `internal/monitor/bucket.go` to report metrics. These files are using the archived lib `go.opencensus.io` as well. To migrate all of these libs is out of this PR's scope.

Can we accept this PR using the archived package, and create a separate task to migrate all the existing exporters to open telemetry?